packages/turbo-workspaces/src/cli.ts

```diff
@@ -41,13 +40,18 @@ workspacesCli
   .action(summary);
 
 workspacesCli.parseAsync().catch((error) => {
+  // eslint-disable-next-line no-console
```

Comment:
In #5832 we swap to use our wrapper around `console.log`, is it worthwhile to switch that here too?

Reply:
Yes definitely, I like that method better since we don't need disables everywhere and it consolidates where we're actually logging to the user in case we want to make changes there in the future.